unittests/CMakeLists.txt

```diff
@@ -1,14 +1,17 @@
 add_custom_target(VaRAFeatureUnitTests)
+add_custom_target(VaRAConfigurationUnitTests)
```

Comment:
I think you miss understood my intention with the `VaRAFeatureUnitTests`
The idea was that this represents ALL vara-feature unittests, meaning configuration tests would be in a subfolder/substructure of this unittest package.

Maybe we should restructure the tests a bit, it's currently not 100% clear as up to know everything was called feature and "dumped" into the same folder. I'll try to have a look at this, when I find time.

Reply:
So, should I move the tests inside the vara-feature folder?